From 2bd1df6e732f842519955ed0d7e2055fa32f1e8a Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Thu, 29 Oct 2026 09:06:52 +0100
Subject: [PATCH 1/2] Update the upstream signing key

Refresh nettle-release-keyring.gpg from keyservers/WKD: same
certificate(s), with self-signatures our copy was missing.
The commands are in the comment above its Source line.
nettle-release-keyring.gpg was in the lookaside cache, which a pull
request cannot update, so it moves into git (dropped from sources).

The current key is needed to verify the release signature with
openpgpverify (sqv), which checks that the key was valid when the
release was signed.

Assisted-by: Claude Opus 5.5
---
 .gitignore                 |   1 -
 mingw-nettle.spec          |   4 +
 nettle-release-keyring.gpg | 518 +++++++++++++++++++++++++++++++++++++
 sources                    |   1 -
 4 files changed, 522 insertions(+), 2 deletions(-)
 create mode 100644 nettle-release-keyring.gpg

diff --git a/.gitignore b/.gitignore
index ee75e9e..a589309 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,2 @@
 /nettle-*.tar.gz
 /nettle-*.tar.gz.sig
-/nettle-release-keyring.gpg
diff --git a/mingw-nettle.spec b/mingw-nettle.spec
index 98a9de8..55dfcdb 100644
--- a/mingw-nettle.spec
+++ b/mingw-nettle.spec
@@ -11,6 +11,10 @@ URL:    http://www.lysator.liu.se/~nisse/nettle/
 
 Source0: http://www.lysator.liu.se/~nisse/archive/nettle-%{version}.tar.gz
 Source1: http://www.lysator.liu.se/~nisse/archive/nettle-%{version}.tar.gz.sig
+# Refreshed from keyservers/WKD (same keys, newer signatures):
+#   sq --home=none network search --output - 343C2FF0FBEE5EC2EDBEF399F3599FF828C67298 | sq keyring filter --experimental --cert 343C2FF0FBEE5EC2EDBEF399F3599FF828C67298 --output update.pgp
+#   sq keyring merge --output nettle-release-keyring.gpg.new nettle-release-keyring.gpg update.pgp
+#   mv nettle-release-keyring.gpg.new nettle-release-keyring.gpg
 Source2: nettle-release-keyring.gpg
 # MinGW does not support explicit_bzero()
 #Patch0:  nettle-3.8-zeroize-stack.patch
diff --git a/nettle-release-keyring.gpg b/nettle-release-keyring.gpg
new file mode 100644
index 0000000..1a79ad2
--- /dev/null
+++ b/nettle-release-keyring.gpg
@@ -0,0 +1,518 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+
+xsCNBFDrIWMBCgCyyYoTAD/aL6Yl90eSJ1xuFpODTcwyRZsNSUZKSmKwnqXo9LgS
+2B00yVZ2nO2OrSmWPiYikTciitv04bAqFaggSstx6hlni6n3h2PL0jXpf9EI6qOO
+oKwi2IVtbBnJAhWpfRcAce6WEqvnav6KjuBM3lr8/5GzDV8tm6+X/G/paTnBqTB9
+pBxrH7smB+iRjDt/6ykWkbYLd6uBKzIkAp4HqAZb/aZMvxI28PeWGjZJQYq2nVPf
+LroM6Ub/sNlXpv/bmHJusFQjUL368njhZD1+aVLCUfBCCDzvZc3EYt3wBkbmuCiA
+xOb9ramHgiVkNENtzXR+sbQHtKRQv/jllY1qxROM2/rWmL+HohdxL5E0VPple2bg
+U/zqX0Hg2byb8FbpzPJO5PnBD+1PME3Uirsly4N7XT80OvhXlYe4t+9X0QARAQAB
+zSROaWVscyBNw7ZsbGVyIDxuaXNzZUBseXNhdG9yLmxpdS5zZT7CwNUEEwEIAD8C
+GwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheAFiEENDwv8PvuXsLtvvOZ81mf+CjG
+cpgFAmhdk9EFCSDYc+4ACgkQ81mf+CjGcpjdRgoAh2OYS1XGGk8gNn5n/7UEfd7q
+Hko/kaJ4yobNgeSyYg5XB8Df7qjH7eiDelTsUXdmLQQskGzqU/prp85mYptPzbXh
+wLYhceH9WK/yP2IBWti20TxzNFqPQKeUzKXtMp3tS75E7klOXq9a7Od/DegeriW5
+AZP7Z4zR9pQNNaI3gt80EuIQM3PRqGfZAaU0l6WY+SL+huNtnUFFW/930M4hAbNd
+uwRZymzMX1Udb7jJQu4+z3C7e0qx/G1UqNolmFfal0R/wfOSS7AptmYRhzAf/ns1
+Lti0mu12WaDpmBvk3Ti57MC1x2qa9O1a6R4YRLyRpmIfPq+h8Ez4vfIhKElz8CPb
+tMNxnKpcs8yi528WLG5JKGxEY9iTMbOtzKQ55YhmKMkq+lkRcUrX8qLqydBEz1SV
+Lcit6QA0I6xwnIH+PzzCwNUEEwEIAD8CGwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4B
+AheAFiEENDwv8PvuXsLtvvOZ81mf+CjGcpgFAl4h6wsFCRacyygACgkQ81mf+CjG
+cpjoSwoAmooT2ZjT3zA/km9iJ9pDEZovgOyVlTSZdohKWp5xtI8C59uZuxuHV9iJ
+igyNWnIBVBr8FjL6Zx5paNQ19SllE1bYxL4J0jw5j0BP0odT5jORkIsylcKHmR+e
+SqJiSMvHGsd821UTagYcJu6emat+KcwnDHkKPjbEoRmi46n5UzIEG+uHv0sGZUjW
+ZshTCQZVBnJj4sDNJl+kCbYTpUs0f2AEPjKH6pBk56vIKBP/bNWs2Q2s+VdA7/g5
+A1N0SkaPt3/+qNslu84qRdIFcqc54stmR//Qa3C1EBxrrT2P3EzzpkHWxO72jaGl
+wuN6utX+7YuNe5Cy5ls/BSjugKMiRqBEAYvFmnbKV2eJS1bqTSR+qTzLn+VS88yv
+dumAHNNOPsJyMmKPxJD08maMCsqOOys3TMl5J+Yz5bSPJQAZ7mvCwNUEEwECACgF
+AlDrIWMCGwMFCRLMAwAGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheAACEJEPNZn/go
+xnKYFiEENDwv8PvuXsLtvvOZ81mf+CjGcpiqbQn8DhvpMTmo9aJaPSSuJQFKanKz
+epTP3eBHoW3vDZ/jvwom7NtJp2/KfS49yc/boAZM5CrvlGG2AImB1hu9GhxOOUyH
+2M5fnzgfHyGL9BKukJJgyePp1Apu7RhAgWNnHyA95i/Mby3fgUI0TNnT1FBmrL+6
+RHI330uTAS4/gVarjzpvt2mC/sYL5IeVlwxyjpVbwUs5PsoxqUfTpyZ2EU+sVJgA
+gcO7cKxiELySchBX31fOej8p5KQTCSq1gM53PTHgdZ07AeuXZLZfoTHWrdBFdyrz
+gRhDAfPZYQZo9HtV3vo/knXCIc3qvA7h7F+CxHJO9zo0NA9MWqagLPC4++JiTok2
+cev0TlOQtPuUCcRISEwvLIgb/tTCmC7kLvUgxpOk6l/wMRT01T/O7GAegxooFlp5
+gqzRLbqzXPAY6VBfRP3CdQQQFggAHRYhBPHSWRqhmdfGCNdVnBzU8yjmG2HKBQJi
+H9YgAAoJEBzU8yjmG2HKpZ0A/1kSRNURlcSx0ZFW3cMCoEA1MkXVdQ6CBsim14zr
+c3vrAP0bBr6s7joyN52n8cwQyVG3Wq/ZNrNkgPvF8i4MeftGDcLBXAQQAQgABgUC
+Wid1vwAKCRAg3XhjTBvro65gD/97JXbvm66kVJb2Bhj/J43j0/5l5tD1vBfZVGUb
+sv+4yDKzQ36jttRJX5f5P4EyisJdY3haF/yA8nyrHyFmFDoqAaV6BXOGRhEKCJkj
+PEHXEh/Z8RbvEluN/Wv/GmKM1Fbpmed9kLjKXkhpnC5KwqEg8ABoJRyQmHDi/LJA
+a4FFKn3cmVViZYO6OsDrgeZwXdbA/bgwFakEqeMgtJVEug5kc2ms9ARAWvJlLsGG
+GSkqZNrblGmHb2ePpJqIerafa+SHhOYnIVmBsulA6nyjr55CC8uQm8o9l49ghvGB
+CNF3r9NXJI18tqJNySdUAH7WTbiDvmDMMCZYenzZwalpczTTsLz4cewhVWlp8/17
+r2JHjDU4X5DOo4yGn9WzeNDDYnKnPof3zvKTe/Y0oRw8xreeVcQvwYd1oEO3vFcq
+1rUlSeWrgCU/tuywFCKve8V8qsiLRYid/Zy/f5r5Rc6Y+3Gd2iExLayEEERP7ZeV
+cA9u4lpx4QiUBohu3aobv4L1PAejGkz/23YJRdyot86GYc1MhWBvlTG66nmsgqid
+EKkKSqtT/k3w5lxFzSNfyk3vXjCWTGfDKo+xKuyCWYW2xF5P5zxh28/zkWkpFIBI
+vGbPfwiWx9Aaqzas96TkanANjyLISp3gCjJyJboMN6Cjz2mIyEhFJ5iPPFL+Pzps
+5zvIWMLAXAQQAQIABgUCV+99nwAKCRBx2PH/Noxmd7ZOCACdgNvX5/laJOK3V8kj
+K5PScJ0USMpzblruoamRcx8Jb/qTQnGntmnRrIOA6KQuLEXq2d+QRP1yFc4KoorZ
+q/zVBACxYIIlRNyZ65SWole444hOaNtIB1qlsNYAXoGyLbmh8FIu8k8nlHcO2hV2
+rSynQpovSu4EYmVuOFy0Ai+E7j0Ght9g5XvswVaqjhkkYEho2bL+tA79pnS08Dk3
+D7awjs3Z1HYrxZ/2WuXxSKMJxkl1mk/yYgLK9zbEIQJa0GsHuRuAFD3AeLiW4+LU
+R+b/i6D4/pg3ZeumYZvmB9nzvPU/PCrrSWiEMtGfeV9EgKIKq+++NH8l7nNKbMW0
+HiZpwsBgBBABAgAKBQJXXJ+ZAwUCeAAKCRDwS2N2CHpewYvXB/4ywSL4+7Fakso+
++EzrYAbi1KP0f1GiTBA4SjdCR5IC01/+fNg3aqMWcebY5Oyeh1gUAnfF0tyPVFid
+4ZKbUbIMDCJK9hd4TYyp5qJ6t2T6hNT69BkNcU+BHU9uA8VYODUMNnvzrrltPvQm
+8av0qcuBcB70+zJdZsgELDS2AODV2PLkzZgXdH2ROpNLKMdtfM1k3JKewvEQOPK0
+OlhAYNcsu95mHeeisv/HHhNZiUlgqNUdpp4NsGYosMRDnXTyWaUS3Is6a5xIRUWl
+lBV50imMd+AoEoIbwNloDeh9cmv0O7+rYCb0+jEKkzd2a4WEJDLy49iQf8pfiwKZ
+VceTuuSGwsFcBBEBAgAGBQJTk7AiAAoJEGC8u09c1/nvc2AP/0M2hydF7dV18PZ+
+fpyzJhWFdBkGQgvMUvWLc1z7wr2rO0xi2mO0OdYn29ZjDhYxP52kqSItPrhoetMJ
+TE6yrOhCeJF40V8sMNmSTQ5r7Paqv+0O3dkYKWNekhpFKPZXqt/cG2OUBPEyDUxI
+Y6jOvmqx978LEKKXUkcpBJe9agqd94Kn8fWxkztUVjKIA4VcjRrlhjIcTFjo1EEV
+VCjwe6BwEUoSUvgmWyl1K+brILgIAxLuYT1f9CggTQVnR30f7vETBMSdlkYnwd7u
+K50gcjDZxMm9EZWuUfElj5SJQloz8DxH1pCZLiSaczAZW3kFWnfuAnuoeOEVGlup
+3cqxjc2Aw4KMVaVZZvnqwY+QN00TcpM6u/vA3L9Oc3c0MIWhFcsS6wgUpKf+NQBs
+kIgy0/Rn2NH2Cu24DveQ/xsN/0gmf+CKB1giz67STrvuuAVB93KR2661z61SLUfk
+zk7ECmP0KkUXzorMl8L5VwExXqOmP8mtX5a60HKAg4x5SDPiIcGijjamq4TPiL9B
+Fduyg1GumdC8Agg0A3qBkKfTbwrrIe7H7pihKB14fDXFSfmBQW3aVaQ7ddwxBPXV
+4R7LuiPN5SGdl4E+WdtW/oQkQ7/m8emNafwL7VSXhLdD1UlOFAqfUjTY8p/bUdfR
+y5uyGNeMCeMDsk63dCzEgu0ZZdbEwsFcBBABAgAGBQJTigIZAAoJEEMATQlmX9Vz
+eRkQAI57bGv+q/MDcHUXpGNNLpNKBUCb53FaMDoQcEUCivQOrpTT8p7fWb5c4WuS
+cDcA5Ap5yrexNRNasufZv+zvz/XGanedEupa3aadOI+1hWWm+PwZW0Zw7CCh065v
+xUqYxNWKTO1K/d+xruruP1oznT2fVI2M1mdkbi5D7sVsaHJVkVRs/FvY9gNNs/sA
+WOP01apydVnNDVMauGIhXkb3juMuQ5KeA+vLxZHXNk7xN+k5CwzsLwkNqy+HSCH3
+YJ6znVgHQmcd4aaitE8ZBtS9RTh5tnBAHhY0fnA5pcCRKiWQfMo4Xfa08Y272DpA
+vOm7tRHyrtq2+thKJ5Wj8m6oBmzLp7iHDUeS1LJqO1LDZxBy5EMU7ZVNryg766Uz
+LV+O5SSLuOf+CPJg/ABAnIfXDVQGUdYq8O1BaxaSr3bFYnxifWU5LfGIhcEBg8fG
+e9dk4G6WHvGFzaHieW9q8+Q64NHGAg0GhLeNrgwgpcgjr3zTD23wr/yP9A0nG2xn
++rYeFrWCbqmr/A44XEotQPUNm8syzYSV3xNAFX8bLvfQ9aYMCT88Y1vFZAj2c9t3
+BfGn5YztbQQ7NLFpeMBXlYClzWi0Gvdkr6R8YKAprV0JlYZlUyc9fiwXXPrZQMD9
+8tbIyE0qbj2xiLTSrtDy3Z+aU/Zal9GACwwc9T794dfu5jx/wsFcBBABCgAGBQJT
+fFMKAAoJEBFFk0rgesW1qu4QALLKRKUVbjseuSuC7OrrYDpZ6nVa28IkNQNAutBO
+00YKPWgxwjUbndho0JffpUWBREIvitgmqA8XYf/MPrJCuER4tgl1zJg3TfVMj21E
+9tqDVDcUWXU8/WEUGeIQnSI+RCUT1T810sfcQ59t87UBhuz6IB4c7vQKjVA2lNDK
+qX7cHlT0N1a3y7PcAUBjHDPMG4groiemPrHUe8pZNUZKzF5oihK4lP9NJqpucjdR
+17gTQnQu0iAjGNTyzxSRTdEU0bfUIZzMQ2lHjtg362gQT76aehsptNrkaNKeSEOn
+lr/69mM8J2PIGhRN77YYM08PIMXjiRxpO/qgN2c2YA9BjxqAaggqn7MgL/AuzMJ3
+fMf1MoIsG7s3Qbgtv48/+t90j9NE/DW63kFiWjtPgI2CJP5Z+2tOh1HpeTIDHwv+
+gn+yhmsvIIySAPNTgZKBdPUZdlIcuTpWMJUrum4ao+WYylZ/CHeJo0wjlOLNEQTg
+ULGPMwzc3jZMHHTFYHAqSKwbLY+KoMTAA+w4HR5nsaSCSsBHryH8Ba8QRCg0iCjF
+24R1YANeKb16f9TjTgsG9A+anTib25yVB6zDwAcrpxXqvcTGtzsJJGFXI9TI/tUL
+UayuP7jrz8TxKRgfyBAoCrVxk2ctOMZaU9v0mTUn5J+kQA711b7erqkjBAp/VYJF
+F5XYwsFcBBABAgAGBQJTZ481AAoJEEgEtLw/Ve77OMgQALLXJWQZ0k0V7Ryw99b5
+Kpb4PUZzKpA+zu++egx4pxrF6kdMa1oXyV5GZ73oWIW1rNuYieYokTVs+odQ9HkG
+Rqv2yUpxTUabHnj/gs8HEIETSE4L+5LTlaTdbk2kRxjYNtWC8DQdBfcEvlgC3m0d
+FfNp9L+qFhgKmg1QJhM0pM4IAQk1UXJscUo8beHF6m/i0Ffc76gWIbEiDzGSAKz4
+4s+J99taQYqUMN7kbeyWgR8tcGfUmHGu4MrRGtjes3g8EzxY0G6HN9H5Rb8cQr60
+9QBBifrUbcCQVkfAbMgIp+G0y3Jc/VPdbw0ismkZa9Aw/dXf1zOtrr029squtOt8
+WKwpmip4GyeB2FSjUpXvr/Tru+JMpzr3F9D+4Xhxxn1vp2XFjrEBIMq0jIXNvtuP
+fYhKCBgMc2j3hlkoWEgQotJAF1O/t9KFCwl80l1VWh0lAS3qzSOKkQSjFKPKYK+L
+D88fKxjcI2yDemIJCAqIui8bOyz45zr63i4BcqEHIyrk6tYmaKmGV3KZRE+SJFfU
+IejgPgFotEkYmMVm7dTZcdxjmgJZAglBlNI5/ZIvH8Nu4KbD8d/6MDxu0runemHp
+7EhfEbUOPjZgUpiBm4Yho9tZC5gfsku5KPlXrXikF0h7qNTcej4T58tutyg8BMfG
++KjUAqVP2IOlIZOyliU7x92RwsFcBBABAgAGBQJTZ48mAAoJEItYgK+sAhjHJp4P
+/1969HOVkF9T09nSp+RK8CZDY1sFFcqWXCLAT27Vhd3nyDVLR25pEM03PDOpK68U
+/fecCMWmdnihakHNfu3G6fHhE44wAViiorlxq9hwP1Zu447Wn4UldvsZ852eQ7Bq
+QT4R8lIyO4674TuNqJpoLLRKQaNIjzERyqp6Hmco734rghENaFAcgO7sjydDobDA
+EJnLDlf8Ph8djkkxspEilo8wu5BluXNcTmpmr8C/uJYZqjwC529zOHmSkBAZVy1s
+1Xns1ZcvA7/h6Jw6vMI7FokmsiN55igpjko1CUDET08SKxldtThnlxxi6gkjtptD
+n57JZSKasIDN6EzeibUPMW9sBOPzbx+y+tTehAHXqOto9n/1RJ9Oy9VXMfEOPv5c
+r3DATSuB/39DLR5WdXx85Ijcj6ym7qKbYrdAxsI2oczCXSYODkE5vTgO2JLakdnn
+HMR5pHWwdxnBhpNYNoNIOSr6XokYmZeH4lH/4rBOcMCVZaoG2t+aYIT1l1xkK0UR
+QFr+U7+ILUdG341P+D0Ef7DjYYZZtmJd5Lsp01pYtsDbbXmEhaeusWQAvxUmIJIo
+r8jHsSzt39Zr9Uda9ZRFmtjMJwmrCrvs4TnuxHHoUyEmzK+eFe3hJF/9dNNs/rhN
+8QqG62fi19NusehnQYIaLt2Kvj6O3+9WmeJo/aeTZQu6wsNcBBIBAgAGBQJTLspS
+AAoJEP1Kkf6BjrHxBu0gAIkh7j5gTvxRNtGComCufJCiYZHwXu4NFu68I5V1GwQ7
+Njc9BZ9oMmCSCe/gSg+OWx8ci984AopbWdsJlWC5k5ux4i6ASZXLrMvtGqvMRdrf
+2ptDxSGUGbAzArNSqU7kQLzCIUGi2ef9uP6hPUQjlrzpIziP4w6MafsjJe/Mur5N
+bXv+Mpfk16dlv8tEKGyTQ8BUw6hVWsRIybPCySuyW6rcReqZRzrZPO+AGm1AIXb0
+7V98TTDaGQDaDljHECqVScjLlQiLDKIRLnOGhrBhOMijbwJxsCBtPUCET72vx4hl
+ZfgYCUIJnyffYrMIzPN7aPXPigV5ZgXiM68AeGIjSHqSYt7h+RLUOL+7cbcl/Ce1
+7BWVdfrNFJGD/6/IqAmL9sptzuCov+inCHUoilnZLrqi8Qw61TAwZpTGmcpNnZEd
+dO3SVINwhZF3KouqA5PPcvhZFTX/MCODlCWBsw1AWEX1E3fecyf+iAa0wve8WmhV
+QtKQY3puVzaIOk2ggo+n5olDnFai8aQdHmhQV4+9LeY9wvWdIFVwrdiiRVzy68CK
+U+NUTVmpakX09GRxznpde5cRFsL7gUy1Nbce9A8EJFdVPaiMAZvbmp455CHzNbOO
+dW+utpXnvJB14MfqmqWMCuX8E1tEqHbbsq2IOvW4Cu2JbMVCe6JJHcq5D7IE92hW
+lvRKfQN1FwpMPCvASx9p+HDMKoUet+FN9DE9+6z+p8hREr2Ss76NXdwl6Dj6omPC
+cQwp0qU5Xbd7dwXa3O+9XEc3ceuEtu9UyMCWHV54LFfuImkoyY+V5CRYsJRkD4me
+rgqxh76utRi+viQIJCL3ZT1qVjBThB6cMxyjQV7imF/2J/DSyR8VDcksDolMQVS3
+Tq0JxZbuqbGVAbqqUuofmB2czN829hKOnqGWJfeotXmvzK74QRlkdo9wrjcKLnmc
+veKLxIUxVpaZQuwvdwsozsFwM+BMYW43pEOHU9KKBcdmjjlTjBJam1tBOXscKHGX
+AG+Tj2Ts2M7dGLTNjYTRSLm4lOSPT3+69ZH9cg5Vi5BWf7cDedQLwBaTapot3MxT
+AVP7hKfYNMZme+sP4O4faoY2tYYJ8RBPFQgQm9mReb12C2Cw8dzjB3uIQQIK3DJ1
+kttk3x1CbTOA5rBzM7ZlY9Xuf53+F3QXm3+cKc0zhzaNmpxX+Fw2xq+nTH+v7P3x
+TG4Ji8qZFZ3a3MqYE70ogWK+54/6YYjv/uY6rQt4pIR2Xfs+C9uK4/B6nS9ejs+C
+6axN4gf/JMIDdBonvj78yyHt/CTtR5ktpeOkU7Ei7ZE+V7nGy76MKy2sd8oWs+f8
+4EUBUcpr3tJKQfN3tFrdcHaSwW3lEqyt6np+D7wkFgvCwVwEEgEIAAYFAlMl3WIA
+CgkQkYFdZwKYKkqsBRAAlWKKYkqyMC1cwNji4kPwQ2iF2+J84fzP0D7/p5dRUA6R
+TxFeRvxN6mir5SZHn28Etf9aZr8MZU67SPxfzLj9VraLW1QalNahMngB1V1bKMVc
+wyV3f9Ox72BBaJtSrV6O1pJMt/yGX60QrlEY9kXNw39jcONSimVKhPvi30KGY3ii
+wZ2OoyDtDMsRkhHatar1GLmAPZLsJi3MIw38FleGK3IMOQc5PbgeXzrlaEOneNb8
+WAMK2WwW+iSJmKzDEjDksCfcpwtm3RFAkgpwFBIwlBiWANssyuBAk/s3/RcisTBw
+0TqFl9B1BTow1FionvEnuwKx4Yo0koWTDzCU0RCWDLnLZ+vzdRR8m4tRS5YSbqho
+S1smnyHz6trv/Vl5dpYNIvUmaZx4F1zIq+vOC/tNLe4KObANi0ie9aRROoY6DOzr
+r2LwhkwZWQOFxRWTA6u6vkQzRZMNVL6HAIIeynWq43GRIZA2iI5r/hN96xgwZ/0G
+18CRRjvB0dgczGKCXX70E67qaAdDWr/zBpdMkaNaELvTx8gNEPjSRnsIGFZ0rFym
+6K3FLb2H1neLoRaKGKo27dl9ZEYHgw5V3gV7UCgDjw6BYOIbNnUamTfCx6yGCeLp
+3IzZL0MBuqgHQnHvwSfidsQNKQuP/iYbQVo7vqcBsMQehcAbQPzVVNAs0V+wRiDC
+wVwEEgECAAYFAlMhrwYACgkQ0LD14xrbgZqMAg/9G2TaQloh580628a5rj48sg0X
+wAFkEsEx7rzSlilKHZajCEZzQrkRy8aGEA4a2Vwz/FOURwxfDqvDheQMDCVqYl02
+tpMHsN8l8KfyKK6NTDJh6Rn/BnzyftsxT+NAfL+E9VTWCIbCHbLqLxyb+jnuXANi
+nUFHkNNBU2khm6acvXRd6Z99nEzosVNmCS5w7XfSKtjRoopSlUw7qpFopZRlB7aV
+PUUtAv+aTdZKkZt5iD1nlchJqxeIQ7Bq9qYasEGcwXRxoyMsVYk021JcCZ5mVB9f
+DeO0Rr+Rek/NPjrWe7w2oYnaG0dpP7GtR+3HwW97YjYOGSD3ck40s3TjmQkMYaF8
+BiEL6dcj6/xvuNpWXXjvt25kcdS7I1UxJ8v4bd1xo0p9G02XM0A3yJEA4T0S91DV
+j0K+sXg9d9R053GFDjpU4PZ5/BNhXF/80fWyEZ8Tocw9UldVg7gV10Krf9OoPInT
+qLRo9fwBmKfPzbZ6SjLOg6ia8B0i9OCbpKuA2bCkuDXJiRO2d0P0adEwixKgE+M2
+xMgU/pqAOBXB3s294aViSArNnpV2vdHVm8797QWRJXBfcRffLzKqRlmhXpv7Z8PR
+j8bzN/bzAc0N8tbWMwx+lw5J7nJWQpMWP2UxQPpF10wTyxjb+LUXpFAHd3ypy/VM
+auaKrKBK7TWWnmoovWDCwVwEEAEKAAYFAlMeiIUACgkQOSeNqBCeYkT0NQ//ZzAe
+vji0VdpjoO+EdQeNHvpNj9NvXt/ci8uK/WtCIQDyUUY7//B4Yx+tZukA/mI4/emZ
+C4G8lXb797KnjDtg6Co5jjIIrDy4IaayUFH6aGTzU0wquFCyMUvNn+yMaMhQvhna
+s6SCL6aEbmVqkpSOCMWhoI6Ry/TufkEtsLjMzGONhCfFBH3ECLXMvYNUc7+0genk
+kXxQAbnQWDYBtTpmgDRJ1e9mDjzLR7tsgzbvwDBJGUHQY341BuaaehIN4hs7FK19
+vbNSvcbubyLAzueCLoZPmnnjeo8bupX3mRTxaOOoUlS0UACcy95MvLStihK/G7dt
+oYmZW3GkAcjJy7HMH1ucsPXGOxvMisUPBG6CKNKYIcYnOdP5YtYiYeYluWN4v4Ik
+66TVQhoqKrITh0dNy4IIN9DyPsifnCjs7Dfu8EeRIdi4i5YM5XCGyx8t2ArthswU
+OFcz/vuTwIrlPlRS0PYluV1WnUYxTxQoP5FOhr7zer3ZXYG2Q4YUuJVEkgA9XXmS
+i9w5cCWCroodKJbTA/ZGLvEqf6egZWeKiAONimph/gup1npEDu4U75OerT8CulsW
+kZ1TYM429Wr12QTtZP+3PVaIgQa7k7GduUgBvhamPRHqXpjgHEbbweaSwtAeMyjJ
+7Zpddg2K9D2+bX/iUUXq2z4eOVY+p3yZB/JblVjCwVwEEAECAAYFAlMLkjcACgkQ
+gtUke4G/3HAPzQ/+MbeNGwBUiiAOQXPDMV49iQnhC1aIrCE53SIY+BPuxVOdvvck
+Fcz2Z8SmDQ9bBy/9rjGdedyihiRgFun5H+JXUErYLyBpTyTJwreV1ICxOmhKuB7B
+/rnfWB+3tlKZECrdj9KWGELX/W4RhyMO9PCbSd/b1FdrLXcmtqOxJt0lnp7H8hLQ
+yDwG78dQP2yBZ9m8sgnvuFd1Cnu+l2u2Ijt2pcZ9j8wFZZutcKHLWVTpHf5MaNih
+uf48N5HJYM/s+uMFEA92oYQ7dj+jIW7gDGcnuDnOSwsumFlIBgdwmJB7+Mxb3yXa
+GwilfbkgwQSURwDQLsYU70RF58xeyL0BY11H32/Cp+KEOYfiMtkOou3RslQwT8XA
+e3rh/ZB7bAiGwF8xv9f6caffhLxy/5Nb1P38ntmCW+YeIwapW7hqkInnG960gDBg
+6NFv1KNo5BUREaDD8VDR3WdjIxGGEzdJLJwfhkrZbnQTkqLjV81qrk+J84bT+MwC
+Ahwuo/dD6PYi+/ZeeAFQ782ynjTL2LqWc/Yq46xbsmAgSlj/mdrwtUObkAm/tDBi
+0p5Cxsi2/htXr0XAPRignkXhniyhn+uNokMQ78TFvgrNHATTb4zHcFOZMUxxF8o/
+2qaUg8Fv1ZVl3ZBWBNs8OK9ZDXyJLKttG/ViM0Bgg53LR/ExjiQDC1H03d/CwVwE
+EAECAAYFAlMKewoACgkQCZZR7mU6MAoYeBAAt/J1n5Wl78kaeCo1tLr9S2d4NOhF
+/Ri49dCUOCKeO6uv/18zzWHMONePr2eL8odUqut3UHi+2Z92P6399Y5OKZNebCwJ
+FmCmWITPTEvtUWYi4hxc5w2r4klSRSzgJ8U2nHrOZX4qqpksNYuyrsikoMKT/zWT
+fgWYUUQkpVe8SlJwiOiwyfiaURLRZ1EP/QX1dBk3rU7M/8v3kxQwZl+SwnrHg9ql
+n93ow5WsJoWhuXzYD4sdJoNMRCJ5aNmiU1/7dU46q+szoNQFFUwy60d3zoSByEC0
+0sToNyvpZtZfaEhw0H1Ct61y15KqoX3IA9lrbcAjWzzXkzfhWpm8DLRtNWKdIMW3
+nudFjZr7CnpY3T4fkoXL1LZEsdzUyWLLaJzoTiO1y+CwVD1CWqQ3PB45yDcYKsyL
+nh0v/MTQaMBzCv0fQ5vBS4s5JfLOfOZ6k3VPJ7mPj9D3XjmRE5JIQmRx1iDIm0w8
+G/QLWqZiF3qTFz6u1tL7d8wXD0FTC0p0HpRAypKj56OENkZjyo+8/+lHJLdJxllb
+CbqXdAnJScbOxnGhqADX8j3lcEgPPQyYJet8zp/uEYB+UzxTFXfAGu7FLj0nVHCl
+CsbwkMd3cmsqTpwXa8hdDsJZ4JY01VmRLhVcVs7ccu8t2+mXEA6OeDgjDvuQCwhJ
+48ccQ6Xkv37TXfDCwXsEEwEKACUFAlMA6AkeGmh0dHA6Ly93d3cuZ290aGdvb3Nl
+Lm5ldC9wZ3AvAAoJEHoGhUIeiZBCXhwP/AyhkNSyg2urwrImOS+ZA4zo8SYg1dMu
+MLtP1+ZBgiGw5o9qbZvjzy4kzB/esIhrgCj1co4fSMRttL9acXvRJd0TkpYfDoL+
+JMfhl9QgZETv7cuhTP9QqSCcuwjkyV5i23TdloDYnK3KorTJd+Ar7rizRzDa/aMT
++U8KQM4cEyVRvHTmphCd7tn/bKDm77NbaabejNmIyqvgeQjvf3+P/6LSV0k+zDz1
+CMuQYLA0kLc3PIVByaO8ZhRZqPeJeaocZrUyWdsiJb6/CYYtR7rRcixQGsuKaaFJ
+EPjO/brkZfh2c1CfHSnfNkoiTRuUN+rWz69sOl6Tb8GFOfsthikvMrB/22E5ub4G
+T4jnmUoylQgvTg6PQGWq4+brHymEy4kFAEZBg6zh45Oq3dCdxEEwBUfMiv4kPA2c
+Ho6wEDUaw4phy7oy0TmQloKzF9/zuwRrubNKxEeTtFUsQTV4dzqOnkKaJYOgYAAm
+ZEFic25PDhCcgZNjkGEWHnv+tJUdNmJuL4+HNDoJWSEDY0J2vytbH+HSOxUBrqVG
+hLL09MVP6/R57cxOn0sTWSle1LeKh5vF9fe0fowba9xsNsGP9tWvgzK4GCvYIngA
+HpCSzrAZYnB5zNTJADHFxbzHGV8BAQN1wAJl8YqzaklA9xQ4ge0vML71jO1Peuzt
+b6rcpeBJbuF0wmUEExEKACUFAlMA6AYeGmh0dHA6Ly93d3cuZ290aGdvb3NlLm5l
+dC9wZ3AvAAoJEJSP1qDhD1Auyx4AoP5JnZHPOCnHAO6jGMMl4xfOoABUAJ0eYzrH
+NG/Ei0yzyyGdKdgiTW/GK8JlBBMRCgAlBQJTAOf7HhpodHRwOi8vd3d3LmdvdGhn
+b29zZS5uZXQvcGdwLwAKCRC0deIHurWCKRWaAKCUL3TD91l+YH//brcU5b8jXz+3
+QwCg3ofHt2gXc2+8IOl4ICpA9EU+K1TCwVwEEAECAAYFAlL/zekACgkQyW6LOURK
+fsf0lxAAuroSVSpNanfy9QPaf8R/JKdvG+W6JFhzuSJtQX7uyEHiXA/0jrNIclLw
+AdTn9W6ByQXobXDopBUCz2PXemoq5AENyOPwV/nsxHmSXc231SQ3JBElx2JBet90
+7RhPp/rnVa9NyTvmSYnOFLiGDieB0XYlE5zeXZCC+tboqDx1uRpJ2skUgH5OzYTs
+qOVaLfWGrknt8KAHYcysne6FjSoboqQsjmx8RNyAqWFqznQ8tJkIfZYr393m6d5b
+aVygEnXdvhd1RCFD3TtdbyuSf6xVqA+1JnuFex89ylCfMELkl10taa3LFurhQvAN
+wXBSMqtnPr8eKY+CronAVqzSLmYmH/WCkWEhHXPysK9DxN6pMQaIL7F3FgmDAvZl
+jClYaRUssxIGg6F4Ne/AG6FYUEHSzea7cP6SeJyxxMPLt/mpKnNPbJOpqn/Rj0pW
+l1EUhs5fAH/sq15WCds9uiLPKXZ8uVQ6ATpQGgzXGw7Wj1FCcSUo12z1NILoU2uh
+LyzbwDYS1CdYVtstQvSsqnnnhZCPLbT2xxg/v3fE93zDQQROrzMQRRMP+qmb+P98
+y1eYIwhnmpF6cJjj+5mpCqH6rVyYTPDFS+ewALg1ZZIeYylqtbXQut21Z0qPPtn3
+KVWjAfJPOpjDavVIRWBpAkgGbI48BR4sv18x7U0ff1IXvzzLdKPCwVwEEAECAAYF
+AlL/gRcACgkQ5m5nbAlJTBRKwRAAls1gKy8sjXW781iRVAw+g8rOH1Gqie7TwvdA
+kP55YvEOEgvMCY544glm48HR6UUZRcsj9OtVXx2+NDmZwGC4tVsKHF4+V8UNV7UJ
+UixkkmDCs1sOoCiRnbXNqoZE36MiDbXpClqNfjg8f0j34bECM1xGTvcpjcukrFHB
+HXDuAA4oB6LxO/mgY9SLfMsNbjXbUzRFbtZyK1KeGolLEK8fCw7Pg08vfy7QWoCQ
+wXFq47iLkDVV/YPFJfZftx72rwCXHssw8nAERbvXEd4ztaA7SZvHZ6vezjPGeEyh
+k55rJfsxgWNe6CKG++96lUliTQecjBh6DFDoiLaUZWhZWSlXdgME/GYbF9MoxM+b
+AIEjGZofyYbnqM94cLm+TpwdGHX3SoHhqqRJm8qi/DqoKvWmeJSjZpOLjCYmn7Sc
+zE0JCEqENG9FncWr/GYhzHoa+OZlCUEg/6y08xP1qxo7yd8Q43tqMgpKFrqbN1k5
+wjw3WJGnuVdQDOfuL8/leDHP9DsMb8a+SzIQY/UGme0OJq2eqe0odxg51xAdujde
+bPaxzTIJJDEqE5HuP1qYSSJYLrmuYuFjlMPsDlerA+dkdF1DBHV2ujQbwcAIVpAc
+YLbFXZIyvBBkUfcLiSSGy3Nzo3ibP0aeqHfi9IBMdexfa/Lbg1LPSFfbiOy8e3yR
+vbmM4k7CwFwEEgEKAAYFAlL3zMcACgkQ39J/9/ag8xIVKgf/SXw3m01Cnx/ttDIz
+qL6/KlZU4q7TpVp3kd7QXx+pDg31oRV2SJwi/hhIVWSdrzTU+AUxB3b432r2oOYR
+UxIAyVlCHg7/gr6euwUn6NrOeTjUDdq0ct0SdJd1w9cOdjDpax6gM5Hwqgt+x5cA
+QYFpOV3Of6k3S7socrvlH6hIZ5sYiS+AvYaMYKPIfeVEDualg/Mz0i2ahC7JnJHs
+yAIMtAzKi99VPi13kPMP1Gtl6CRZAzDlsm2DyOwsGwbwPyfVIjSJxvfHbN4zecFj
+vrGKDFei6GCBIr1hqAqp/7V7PjzOs2FS65OMihnac+J/1/zGd0U5GmTcbh32W5z9
+wa9FeMLBXAQSAQIABgUCUveVrQAKCRBbnqFhZpDPlHxaEADK/1Yqdvt4fPh0bSuU
+oGLeBP78CAN05gltbLUpr5X4wx50OI6JnejC98xhoLzg2z1pmjHJY4LeY8eR8xm7
++mPiUOLe38CHG9JlNOjFBZ5fadSZGJL1B+h7QC4oG6qsepsoGyOvhSna9xM6vuPW
+jDD3EBxYflt1ac8uWt7xPTgK7FJqHobUBdUoViyaaOLQ634c3S4P4nfPnQeZS0gV
+e8iJYZRsL6O6STX9N+w2xzMPCgkuYILQOjWeGU35Er6lGa1NaZb2gWhnbo4uw72T
+EuodL4EY49se51C8TuKDdvVHRjN4Dg0PttWzZDHqtGU1Zq2vYfV7T2M0+dMVxHRi
+I/vLOXRk3GfXPLl3XWRMxMLH0aBe1dY+Dltr0SYr6d/nZKpGUKVG6XLsJSZjsHVo
+1BpRYoTJ2PolbTd7ncBI/dHlRf02zcMylvrAGGsaqK/dmXnO/bqZP6i/9hfqBnyX
+F3W7ODU8iOe8TVjxzLhlbpE35VLQfyppk57A1DWGUfZ3dYo4V9DT8Gp2ZMqdjA3p
+8JhggMbfQcOFuan/wC5cOw5GeLTYjMLYelMmGVeP7TUiwWDUQdb/WzUqF7Zoo7vN
+b7wJeXbxSoWK9JwqIPr5W9y4ei7/IGkSnymQ0/561RUW27dNMa1p+8NWgOF9+qLT
+NyDIQB+wkRrRRcLeX2fUOcgZhMLCDgQSAQgAuAUCUvZGpgWDAeEzgHIUgAAAAAAe
+AEtvY2Nhc2lvbkBuaWVscy5kZXN0LXVucmVhY2guYmUiWmltbWVybWFubuKAk1Nh
+c3NhbWFuIGJhc2VkIGtleSBzaWduaW5nIHBhcnR5IGF0IEZPU0RFTTIwMTQgb24g
+MjAxNC0wMi0wMiI4Gmh0dHA6Ly9uaWVscy5kZXN0LXVucmVhY2guYmUvcGdwLWtl
+eS1zaWduaW5nLXBvbGljeS50eHQACgkQJb9ITwirSEndgw//a1QPV0cT3omQhFP1
+2/wwLdoVlEaC/oeq92Yxay6tS9/xcRt/kxm9sFGXauRCwOQSioTqPVDsy7ah2s5f
+cVBLm70vVXYFwZJUuSxMSh3LYV9tchlYOtIX8edDf1Q5G2nWxYHtLF188wrYPPRk
+V37hTmLD7i7M/Kc8jafTlK2fmQIk5RcQ+HA7SxSWJZvgHM6bh8/TQF5spI8WxL3t
+wz7Wa2hmn6pJmjcXTzE62s5eaP+J2KV3QaFmEaWal13F/g/t6V5Y1bQ9po7oa9Ge
+7hwN3r/e9ItEY2QlgjVYd805vCmcqOkmSggcXiHvKUrzeKJJLGHlk2vVcC1s5ITO
+4ltUjBxihV6AeM9ECVQuex67Eg8JV0Ro8eFrFKi6KEY5XKwG1T/ROMSK4Fra79Eu
+t8NPa8h4u+MI6ShE/5+KqY7WokaGQZwl6micqouUpevLM1kpzplYMLfOXKJMmIqm
+EiLrNM5GoHIB1brGMfMrL2FhJlAgdQMFMHV1JPuiuDxvKHMy0ptXZJEYi15u99Gm
+b/l6DqcYFJ3DOjJgLE72cx4yyibzPqHI1bHk6jk1n1tasSpN397smNudhZyxSnDp
+EVrcebNX4NLocV3UTRBPzTrIAmglQ6WF8NsVoO1xPrqNBuPvXG/9waCWWvScvH3u
+Z52hTjF5nV0RWDaqEcunlq+kU3jCwVwEEAECAAYFAlL1AkkACgkQq6HLMOmXo6+F
+8Q//Ws8xwtcPV+T5GMsly+N/vO2CJBJYa4uIPJXvqvVpJ11OCVoneJ+IDZblPW7F
+yt9X//niNgVcisQVzZfwk2+90qvE6rdGJ6fAtMXLkSIm/HR/8djG/dj3mJ/Fx0oH
+ZzLIxWXP1TV0BhNdvkHddN2cubY6rIikj16AzUtvGXq3v7EXU9WB5xzKBScDzPkr
+kw8vglze0VaM0/IJywR3oDxIrGTKu2w8f8E+FHqwRlhjxijAJBaETN1t21Of8RbF
+g/AO1L9SHYgehNpKn83ZUra01wItxsG1QmYWWLKgf1JgkAccGl/f/ckXpNjP9uas
+u/Wmba6RqTBCXPsIgbxHa0M1qvQtnnw61XiTOkN8TyiVk5z727im14811KYW6HGe
+fjPooefPSo4v3lLZVvlNoznURcjSYz1Yo0vBVN/ulDcZk/Be4ByIlaTbfWL6g/fj
+7FUHNH+ULEThmGd9XRk8nnu1BvF3Sn9ngWFeYuTc/8/V9KXmrHAT98yctU4Oq3w9
+4TIICC+eKS/EoQ7CTjll3aKsMxXbEY6gKv0zXmEAWRQCTEev9b5UbCem942nAQwQ
+PBENDWDmS+QjBbcVOBw/Nf07dt/PT9uZJTkpfCLMJdrV8PPOYbxv6Q1hhyzE3i1+
+5/j6MZHplYsjLYPgp+za61RlX1aYm44VibVSPoPGvLz76TPCRgQTEQgABgUCUvTd
+uAAKCRCl5JmPpsyF+2FPAJ94GdVamtip6qD4VGP5gmsvMOpTbQCghDnaG7dI00yY
+HiArAVuHqs9W9unCXgQQEQgABgUCUvQNRAAKCRDJ8LaXneN+1AXXAP9adCRfDSWg
+N8p62AGtCOjTHNtuxZayYHD4OcrOoenN4AD/f3fnuR/uULcYvdkDuHXDP7ob5dDu
+qMgTNqBaa5nJUe/CwVwEEAECAAYFAlLz5z4ACgkQ9ogekIipH6O4uRAAtisA/eJY
+WBsn4F0m9UjsfyO42BTzRyGW0AfBnnwLZowiReGmKQ+2xsNhXq3Ps6bHom5REBJl
+PVLdzyCDoKnavv3V0W5kOcwJChMb75XrNrmedyEkkc/kt6itD0cnRZKwEO8mMACg
+aIxi6xCJqVZfAWTTtkomNsQK2Fb1CNbCBvM72prr6++uCod+iJJUvAn0L8zhwN5m
+CJsACl9YtKfVYVaFUjJNZr/zTvqeR+/WZiOd2LAScMlkNMTLvKxo6CrPGLELSuUm
+ysP1OvjE1tQwR3LxuAu6QrhrUYIQqRUEBpD3V8NwbKxfXWhQjnL+s9JJbEUmW45b
+wN6j4s4V7VmP0UdFKGM3cX4rFvkXFBeQtboo606h1j+M+osimNH/Oq2Zr8THoh+u
+N7A6oqILpzMPw5WNaSt4QzIIOQntbtAjcQGXASZGzdG5ZDgjpZ58lvoTcjQUcXNU
+60bI4QEB6UUhOI94YXrCr/PiZ5pzJB5qip6BO3LBbrnneFnHo242BN8pxIJQywt4
+3fCvgoNclHNU8yWj4mA5jTitYwicMuTNaLENnNKFFXUpZl8EmvY1OjXFbjKI/RZQ
+ge/8yGXppGecJU0T7hS0bZ4o5so1vlfISpfbpX52aIf8Fc5MbVJxhIs8vbCvcI+f
+I1RwVtEcniiDeF3e0P0A9M2RLOWf/2TwFybCwVwEEgEKAAYFAlLzsnIACgkQSCXJ
+CkV1iiHMUhAAg3tH7PbVG4HYoLJNLQEkQQlVecd7LhzlegI2gQSwJVgFnv1RYWdN
+3acj6YPSdmS2t3iYSyXJ56S+vUoo1Y3HNjHxvQh5FR0rGZUlEQWkiijrmGo9K/js
++JAQ6RaNVwbN/W3VyTHnsAst+mPrx8LH5dLrG4+Uqc0dcvd5SYCXZ9TEWcXsE5zo
+9WEAbupHr0a4Uk6s+WDhdnmyM6gC0v4l7tUOyF8zuurDLgFATK3oYbasV4EAb7vX
+WIip+nUuhynkLTfsi8ZJSfGPM9m/pAIbXSyDxC8k7+47D2zEGwNKBMTVzWcu60Mz
+i1mEcNzwpBJw+aX4S1e7+CbC5dJ2+RAWE8vTv6nQr0kizJrHJQ5onp/vWWsnRTfS
+sSYgDPT8zdp2mWPTGnYe1h2s4W4IyZmTqNjKMRzFCVavu1JLbxYGRyjI5MG8xKsA
+vXvQyGus+GIfGqQ0vBQgJFIaW30KNwoLZ/a/kWKwTixZCWly1hB4n1uAnYnpRmOu
+HZXwLsXUw3cGI9E+1txj6IXQKGUVfHrplEUIDJ+KUvKqug5FyiElSnLHVHdySsnM
+pKxfboWjEOd9LxEbsXgcKzeHUqMAOOoyu7wpsOHGSGaFWECYPxCxSMWFqKuLqd5g
+MGiKI2XPL6y0bIy9J308K4zF0+YPhRZWWgCz+5KcyJxRypYCxGg9U1PCwFwEEAEC
+AAYFAlLyb6AACgkQQfrR1SG5SrlRmgf/d75e2DUJNPo/led1jFP8TlYpRIEtm3/t
+hHM97lZjuCi/faFeTt7tOEYZqZBy9Sl9wVoapbzrH4+L7t7t7UDZVRoZHK+gJFmy
+U6zn8+wa7r/xBeJCMlb4RS0SKKr7GH9yL75scMiCjX9DNUCWk4PBiUSAS9tX433t
+NyiXQiSn7gdaM11fKfrQlg9weYsTpwQA664qA6vR65Zrr/VIg0SJO0CGqKbTOxYz
+Nv+vLau1nOQTS2KwOr98MPECapG5OuznXJGmYehudkExPVwMO42mVeqH1DlhZGdx
+6wNJARjKLLUV9qrhO8DrB5t7L/Q2znpiRfjl6+0wmtNYtoovNMjOAsLBXAQQAQgA
+BgUCUvIskwAKCRBqbNW3ZWMtOj+9D/sFICkyrr33v0QlbFKFY7543IkdnAMJo/XH
+VMOfeVzezgDR5/mpjrb8wbfLXEpYDqgV6UmLQzBwJlOGtHOyU+uzUmmZWWsBQh7E
+JDREW5OC21XIHbGkIwF4i5iYXJFF0KL403eYGOkphvGg4HPS5D70IIZlZAhYe7/b
+U72A3Nl41U/OZ14w4uDw6EXo+CraNkno6AzmRg/jKPMbrvr6C0k9rwpHVDoLpppY
+EY7iD+gcI1FakSFbsUxrKmlYJgDSHsCDj/IYoizBx7W7acykxHUPwzGmMxwPsNYH
+nMHIyc357erpbDo4zweKK489gJ94M7pvA9Te0x1PEi+AtgRc23kqIiXxTHycRzJJ
+5GbpeVoTzbIuQfVGP4O1/iOIhJ8jOSCbeZIK8RU1n8lDt2TSRZRL1n3akczsy7BA
+JAICGBM9R5o1yO5Qi78lOr6D9FWGCCHHyZ0508wypy5luZUTps5ZNy396Rk3y4/9
+Vqx/8YQ3pTQTC2dqDXbP9IOPPAs3zA+fPGqZEKCyUcjD7cxfinH3vQAuxOdo6ZBp
+pF5QeTTL+UyotRetdeq5p8QfJzZ2dvbxVOzOj6g2tA6iiyw2LtaH6ByEGo8MLZhD
+LiAaHLuJF3S/TOg3B70q44EjVqbWLZVRbNdeLjCP+M5hsRSLHXrAfww2qadD0049
+UHCDXZ86Q8LBXAQQAQIABgUCUvCdhgAKCRDbPna3qIpeL4ZAEAC4R6NIoHNwnhQP
+bUB6Wsk6bmguSoLjMSjNgJJPqSrYc3E9g4F4oJtLkZDX6drp8DuYyitWUn7MeAXH
+yXjnHViCW/DOq++0Oo21k1jUjvH5OYo4KMC735sVDKYtMzECMiVasnVYEONWW1Gt
+/P7ozYdErNyu6drE/ocUPklIJ+RKTS/EZBgqiqX4jWES71TGPH8oT8UtVzris5KG
++/0XPby51uV2GOKOp0xeCAld0+cnOVqZDb8ptiyo7jnj3Z72548It2G/1/jrJ6ks
+vkiZojtYvtteOouj6VgPuOSg4cKQj4ayrupIA+/t4/pK2lfJg5PLDtNd7Z52PxLb
+3J2URPyNk89UMlUEcjMgvZujBh3zI7cl3UU4fHD9QrQvnegFysavck+bUTMbU6NW
+KAWxTqFWU1xHIfD4wnoF8bDAUGCsLCwfyOTs+sZZD5CJVp6XgTMrcg4gnhTFnWeB
+LnFiQBxrPwdOYrReROv5peODkzjp3iPMdiOTHQfAcbs/jZbpuzTyr4gw5U1oWfpy
+AEC/rui91J0UB18i571S4Ai9SuF4t8byqpo6fxhaO/661HnDcomD2HRrlG+jfhKm
+wwF2NOxz38jDDblhUnNmBVU/EYXdP9dfKRn3w3iLUinsNIbgGIEj6T+rVi4gEauN
+MxqdaiqvV7uc6trJB1eBrOwG9Bo6zMLBXAQQAQgABgUCUvAzqgAKCRB9Ks2vRlPP
+KDhrEADArFcIXLsYbp3bmllxtyAmqXntuSt+LAwBJZ8HrLuxmtkDHWD7ByWf9rsA
+zV35ZsQSmzIXow5dZi66fEZ9MOHTRYAnEaNb4pkJcQbGfI8iKMCvhaffp1rPcj4A
+/kDcufMSWzGpopqcOF6i/o/JPnpBpExoUKFaBwS4tFr2WUtM0kjTaOf8AS9o4xCK
+9+n7NiQqkHcZKMlmSTrU7e/YK4osaJOaBTOCVpLSuherUY0yXMa6vZM60JBLVu+c
+ApnJRKGX0m9UvFDh+9d06sPywCki2Z3G6pIZo+TpbshdTgYeda+oEQXMJS6acd6O
+6cZMrqKTnyNWRIOPi0aM3e3Wwa/bcwITLsgTfmSE9BDmCj6jByqZJPKZcXINfMYT
+4g3pqPHDTae0jLrZTOk15UvTqzKh2tmQmE3Jw2nHksuTCzgd4CZCQwtg1ZSGV+up
+856L8fR74Q8Q2MhwH9BNJe+ET/vyyREdWLKln4rL8+nve5ucSU9PtCkx3pFbMkwt
+erqV5uHFB4VvtbIJr+0Zvzh3JFlyBD/oYAPXx2oEGriL6Sfh7O4BiybQLODZeaIW
+J6AcOPwuH1jlX19TXLM4BfU8R4WDbQkGxmXiiZfEjrh7vighzxty3J389ud+visU
+c4UuNbVNSV1FYr9AYAgM2bKNa0UN806zEu6grzxQ9vG35rHy/cLC3AQSAQgABgUC
+UvArfgAKCRBZj5Wq9smcX0uuHADI3cFPUO3hsgbP8AxlOmfnE0XTXX9lsV0Dtuwr
+FPshKznTdg4tQqtEj6DL3JivLP20TONK+4QZD6j2QwvM2W9d5oeH/JN5nWXchjoJ
+zIYchjbO3nonm0lXxGaWic5iaM3nmnOVfDT5+iuZxLrUC7MiCIVRF26/VUwKcN5e
+Hi4zGFKzr4Zjp6kow5wa3EuI2gXSG49eqRpiU8MjfRUWB+JReVTxyBC9nOz2L9e4
+uB+FkBcvNJ8jQjZb/ti0DXzBk6uPV74gdkZmflaGL6CQvDuImnsBhQm7VjOLeIyw
+Sg+SO8RsTkJ71QY1497IvwErF0eLQzOJgGVKYtGD2XhzPgzyI4uuzA1yIMA1b+CU
+TvQrroAES9Xdn0WqzRPL07gPJiqApZXAoKJYvQht0XAn/2a54m094S0/d/VBqre7
+IgbaXIhCqGAvlbWEyhBX+36+gWBgOg3ySv56WCo5JDYjtabVDuXfB86P5nbSC++I
+6K5Xlygizl7tq8gjJp28IY0b7+PigUVBx1qLZS+SqJAbzDcmL7gMRQ0BtBWqbmQ5
+2tBiOnBqxnZVUW6W4w/w7xcxF2hMypLCPVUrXD3hT3sRgwg/CW7VgVn8Cu3D8P2O
+5gHX+kwqHJOcZqLS3Q7tO8cJROGphhn1ctmDvn+LTnDV0Y4fi6bzvQAvQZ4M6p0g
+gTGDvwCEXUtBlLl1KghgnTdF15FkCSuarhCAC4OEJ+Jor8Hh7sPUHgSqoCgPZLtw
+ANXSUXAAm1Xq9546oRcVK6rrJ+ip21EWIoHczrRJN0GoJQC/ACwtdzMBP/3UZ28k
+9CeIFGjA7OshecHNGMovLq9VWY+UZO9UjnramgB4JvNJsjeGOKWVgt+aj0HRnA/j
+tdhAkjzGNC7G0U9EDVaFXLr/naq6Fizwt5Xa7Y+sAwtv13ZW5ScqN6eu0xQJV+fR
+OhhNzZ+pPgLqHcgPoWtfIFx9CkvTW7b66MXpafg1Z95RxKZFG6hOgkLPp24/N0xZ
+jclKevrDm7Oor2nKPEmyQZbMoXYO4qz9PAkv8NmAQmLGAB3NhrYaWuXcQipBFVTP
+j/7z/TALv1RbGbOW/IDUU8p5O0k1uw1Bw6CtUk0NFjlqJ69wVLgamqmm/di7ie2x
+8DNN64LEyFVTD6zfQb4KYS31eMwQX0nRj5xdyg6qE0qZfkD0Kt7IN2rwUAI86ocL
+tyUETcLBXAQQAQoABgUCUvATOAAKCRAGBPSQykBVvB9CEACLITF17WyU8rXc8jzm
+DCiWWWgBOr5r8jJiF7CY6QwWrvTqNh2MBI/N0Ej015oc0ijRv6fIvmvtLtcjDe3i
+emAZ7olTuHWvENGMX4QnVJ03cG1MDNiuqZxQbDckHi5uAyUWHqhFDlVeCr3anA0q
+YE7R04iozYi0Way9yw7yhiNd1kQ8gDFsPtQ/J3s2qtL3uklIGvr7XBBK5mATEKU7
+QQuLuog6pek7uC8lyOmFSN57Q9jpFBt3yGJEtjZDIJaUpCv1X+Yo9IXsV3i1+u8Z
+NemsTld1EZfNU4txLasJXL4Xsa7eYwdRGFn1zjglJOS3lMYBhrlW/kdsMX4HvH1f
+JJInyc4X0ZQ92PIpzjbaTh261uuXuxbKVEdcbvAsLhzoLdlt9lqbYrZ8CKDVOIxv
+8Ia3YvJSieoLydTj2enhq5pG8c4hZSIzDqoX3WJ2Cw/9paOyBujLHtSYanjySlcI
+/URyta4IbEjMlBHsgf7KN3AquUPRRw1qIvbsEMT5pgIShw40Mj3hsIqpSVA50Mjb
+uViY6ilj0TTHwqCUA7UMFv53EJ9728yviJcYrs9o7fr3gvl8YCrNrmeUPJEPz5i2
+1jthgufVdGKoA8uT0mzEuJRYein4KOh8Q2ImFb5B9clTI2R4sLHe/pBm5YHDK+53
+poMN1hvBy5tM32mIAZV0cwgRfMLBXAQTAQoABgUCUu/Q6wAKCRCpyGyN066NOiDK
+D/wPtMoGz3VJQABzU//38eR566SAJiZ215Ahq7OUODM7ovHdbYWS5J82jcQmEmm3
+5slgMroZeicClyyPm5BD3JNLXVuZsOfj/wl1Ip3qyO7AdX4wWf+IuYBOTbveUK3a
+odQwWIJbn6jsfnjHNGo+Y4nxU5OYAefqOHyj+lvKRrpPTFOY+9bvKq8PLe9rAocs
+QnmirN4WSUP76XOMNmy0AxKqo91D0+aX9N2g+xyH7NYEfFJ7G3R+4biyErnev2cE
+MzrnYFxj1J84Pf2TRWGsKrGeeEnBttQZGic/3/82uhvmHJb9lj3roHEo7PWh7B1J
+kE/XcJmTYGd7WAD3W2bW9KW2vBkOsYKsOtZNG9VcCupXU/iI8uXrusd5t17ojXdF
+VAfFIzaLQUGkDJIjG65kzqFiWCS3cIqW6M71xq1izAUgojtA7SvOuDdLDcRkyYZD
+gxcGPAqAcYi5NOVMwZG8u4BiJJiQ39p3d7F15z8bniXhFXyU3kFmi+n1LNYt3bf8
+kuqMlopOxl42egzX/S7JOdY9GAl17/z0dKNEFgHF0cMEfR96oNtx8ohjMMTP7exG
+Yf59gdOCXjFJtT1FRvZ7VhVn+jtKD938wUoPjZBzT9s1ZXefX76JS77H+rPn/xGc
+DfFcfz+dHlYnxo9rjnI9ppHYQkWKrYVcHGW6XxlEzci5NsLBXAQQAQgABgUCUu+n
+PQAKCRA4dwaiaZjlxKGgD/9JcFkHJAAuP3MyVvzz5afT5IUHTGmeDo+KqoLjBz9i
+AUZeWQGZG43EHnd4HqwdnN57uFuoPkFjODAjqECHwuU8o5am7GWKZs1zAQgt9nXM
+/hGRJ4zLRHZv3FbBnhFuP7TTCbvS0zoxL5OhUhT7wRnmtNql76F3RqC7jA0UxeFV
+W8BF46Cwo3X1baHMEX8xQ/bh4g6VW2Y8ErOON+BpNmSPGs7OYrsopS9uO6QanOQk
+3VWhTVZryxZLtYKoWBzZ2Rzz+aQdct0jOouN+ZL7V3Xdrp0TjoaAqYx9n98gXgq9
+4gR4j1OtmE7oPLrGtvs1mOZcotW1GVedaMlwVGXiJbyovJgsnAW7xhsvp0++SOIo
+uviqBjEccuVKmVk+CWNW2HetCTG9Xg+Npqi959hHaDRMiePPWlkIOr1xclydDFr+
+yo802NA47V4v4Oc1pcd9tr19TBMgECrFChx7nMllu47/DipCTnBnysjOf2jWoTXd
+UNTavzPQtryEmiOM2jt/Yw/RSzylnnI6ZAS/LrcUEpFWZzfz5+9bft2hv1qs+PTj
+neTj2KEM++gmtn21l+k090OniYuhbWGdm70GuU5q83/znKr16gjYBNcnt5Td/j9n
+z2i1HClTCdUPdYkjZe6P80+i7fvsAPkrCnJDS3mkFD+ZiSLSz6ogdrOFw+OF1e+r
+vMJGBBARCAAGBQJS76VCAAoJEIE/6Dp0l0gkeu8An1fcaEd8+3GMAk24qYRsQssg
+X3IYAKDQHsgCC4/bDgXO8QEH6fDKDRh9H8LBXAQQAQIABgUCUu+hmgAKCRB4kQzE
+U7yhDhRpD/sGaJwM5uV6ngy53C4emTE5NbkWF8e+UhG2GhdXrEhIgR1HkPY9X+tZ
+wTZH4xrbzVax1ejeNzg3QM/2nCvxQoijzqaklCu4YKYut/cJmRUzMywRrSdx42ty
+4rKctLSYlwM4CsQzFbqpJWP4szW/Iipnj876zT5GMRvDGyPLrdAG1YYiZBFn1gzQ
+CVbH2+gFokhWvyRTubt7c2X9Nw0TX/o+1yB5VjRBEwGWgoE56VrLosomGZ+IEdRd
+Wr5Ax6iProEfrD0nY/zRB+AQREPQIaoYVJA1MJOgyZLgSCJR6Eb5mERr7aIy0jQ6
+QKbDPvaOJq/UxIPgDh2bZPmpMu8MpsEuZgh4yo/GSsWBiJLz8632j8ReGYCaArt4
+4jGkGfv9qIAnEYmnxsJ6UZ592PLdhEYh2rgDJ7fMwcZ8A4AowwChy2ezRx+vQ/dU
+lkNq6EF1t/qEzeY2IJoeNzV/9Wpou1M98cnIAh2kuameFIpYarAmbpsPs0V3IBhw
+qLthaRXQkRas/yO+21TMGIXKTCUcZwxpdceoGu+IZvaGdXhem/GBdU/ls2OjJSVt
+5CwioyjxrZsRS3cSK6xdhHCWiCOFH8+LSLc8663pySqIOEH1lELRVhAzbZV/XvAm
+mRcQbzxwemYxuejlonFnYLFhNRyoxJzkCOTEf8Wtj2GtjYVWxBXy48LC3AQQAQgA
+BgUCUu+hcQAKCRBZj5Wq9smcX4KzHAC7g22vj1BV2wD9Haeflvx+2QD3+U86AZoK
+OObNOXaJtOX26ylGCPN5mdfchlnFVWo6wLSKfO00xXhj5pKwhF6jjSmGuLxiRxmQ
+OktfRGi2rHi9E9w/nE/tcnA4tv8vUY6gRzj8N7AdSJRk180cC2ekMmgJe3btes4W
+apcFRCZ6ACuvtGDE0CemtgyXVESJfWi3gu0MG6H488s4vfiToOLV9Ia18wZx2Jcl
+yVWVXEs1LkGbw/CfaGWW9RjTcDpdku1+FGaBKrIS6V16MWhuETnK/P6UZhGFEdEH
+VmqeWU3lt9Tck2AK0bAr68cmbv+GXYrpihJak4UVnFyC6M6X3qg4cdAGZ59cF0wB
+C5tWBgzW4R7QVFEPY3Czr4Wz74BeDm6Rb7E3nO3SNKKRIewNAHEaC7TF0rBx/i/D
+QDgH8V4vBQ18gNTNrZjrtHj0AvMQ8/P23AzMOjVAZ9pbPz95t4q1c52aCvN0o6ZH
+78L7RBIx4nrix6eU8MWosDvD9EticgGM94EGFqp84WlzHL169GjLkTCd8MfOEd7S
+nUWa8YtvzrFJMbM7UU1ItHhGQaKl0/83BbP4PVLEtrdKjMZ/EcmcoD+AzfOttxzC
+6asy0O39ya/tIIFE81OLzh3ZXb1gWowuOMqk1YAdVDvEszKCiSCMG27ZFM0BviBc
+bpCK5Hl9pJ4MdUeBDaNd3qjOjGv4SIQmXXIUhnAcPU+EClbWzApTa9ScFjQQywBm
+R1SoLQXku2+df9XaM+8j5hCfo4ITCq6bd9P3YGQ1kbiMzyeJPw6Nja4cow6dsd2s
+bqnVxKUIzURIYtAGVi5HNMkmbl31KTrCoyhWNYyysMoTWo4vlWk98cvSZp4CzNeR
+JFn3gh3tENNAl8QcD87W84Xwr7gtFf3I8XcJ1ZYRCkI5RCKjwrZxgZpu49YwMAlp
+5ermla/ZiPq1+ay/2KzFqakVBht/ebCmwn/Jp/a8GnS4wlxRxJ1YibA/782ymycu
+ghLwc4ewRVlZ8xttQjnOIUCAKpOsmyYMj4Z51e6b0nLQoiHeRXKxj0IUmnFwQ6Y9
+n5ces6h0ZXbb+KIL7F+NHZpuTZaAOohlcKKt71Drtw8ZOEMv0xkD1od8ukTuVamH
+brMDPAehsiFZvbDPOAJVdl60jPj+6h7G34kaPsASqanMPYOYUJo0YfyBDmYA3Sw0
+h6X79baWPMJeBBARCAAGBQJS71qkAAoJEAJasBBrF+oevRAA/iMVBtWPNJjX7dvu
+6M2QD2n7h9dCf4GLAvLXSsVTsG0JAP9TBZHCX+7Dj15fv5opZgle3m5nSGEUw/Yo
+xaI7P8pe0sLBXAQQAQgABgUCUu7WsgAKCRCL6HmwKHMeHM/tD/9KyMI9QbgAjzqe
+nBjyCYFalHJTYTRYYRserhEp1xBkpwkMoTxm4iuNkVZgRuLSj0N1GSfEr8OGdwN6
+b9hRuKRN1rfTVbrZg19M4n6hr369fNuMWOnhGY9wZm0gML1JNefn8p2sY+q1QDhB
+hbC/hZF4b1HiGMKhczZq9Bvo0XRzUjdHvW55rtxb0grZxUHfo8M4+vbvxwqxpsgd
+ZNjh8TZ4H3ZqH1ClYBsdP7IBTFhFuUtLqyUdi3mvVhSzR6ioIqjXtJBZnzchoXY4
+/KsJdBtOWn3YAbeZoNws0mTUmgIheLUUDqj/PcLpLnZxbdJ2u8VhJjbX4TQ7uhUN
++MfU9VrYpO08waghSbdsrCkONjDd0E/FG9GWBH52AIUlt2tMw6fHZOlQdGBoxzS4
+RFB8SCFunVW25DJyK/6R4UP4kDE4qv6Grt7ck0lRYj38KtnplrjWfYRTfJssf0ac
+fzuw/wFJgd/PNrAXewkROpJwzuFCKpbamj0sEmLfplnCxHilSFhVWKr4rY1auic5
+iF2GjXe/fU2Vxvd6l5y9NbTD6msjc6kWHMS3XWtGgE1PivUUrVmxH1DXMjFYT2HO
+y3dzj8ths/ima2NQcYTTYvBuJlHCprDEVgLSWFJD2kERNBeG8WPayA+QKYe6jT2T
+eA+hQ9I2g8KMul/RAFazXEDhqHa9UsLAXAQQAQIABgUCUu693QAKCRAxfqbGmBxO
+SD+LCACSqW/pB1BEaEPkxmJhkZgex06SqOAJza2Q5exmN/Nk1Wh/ZbTLDAgjq3zn
+tQVqttrSxDCItOWB20vsnBS5xaQb7TwxV4oLrhxmqUktDXyWTrBE3qpLrq20tx5L
+0l9HbClvnYkxeBlojGQt5m2KwdKUnTWE5GrxvkDTRLonKnL3nXDvlTOJQV3Dez6w
+lEqBYYj/fntw6eH8k1Gz7zYW2hrNZ7uzhlI4FYEThQv8mXrxp1TW8E0ql9ub7Cv+
+X0ljbhC7cXIcOcvtGesXduYACwGgHcUeiR85QdyFVqoMrMSlUqaxEbP41Tt0p4KD
+y5slR0OIoSJ0ayzFCJjWxZ/sYRj9wl4EEBEIAAYFAlLuusAACgkQMgmq1pDTmtG0
+0wEAoSYrUyHKPgPfb9wKo0hp0Gr0E+/ulwWVnESSH982N1YA/2g7JmcLVplU2SMD
+c8KJd+xEY0ZF28QMgV8K7vb/4r8wwsFcBBABCgAGBQJS7nqkAAoJEHSCZV4wfjRS
+FPwP/jfwXz7m7/seFazV/3mEox6g6vk27RROH6FOVMTD/vnawd2Tlc5sqwz1P/We
+y91ACtreFh2o3owLuhcwz2sPX/3NKk/UGM8tSZQuxkQHyhn8dtSI3Gwqnw9YDcyZ
+Mht5rFf9oOXFH66jx3WFxEtyV7YBhA+AQobUpGF1ikfrZM6Nv9AWw/uvFWE5Bj30
+Q8bYHNFwxQFiaJR2g6ha9QX992InYGuJ/T9+qrwbsNG2rQWAgLEjNqEWa2ci/t04
+SC1asDECqaejDkd5eIG/xr4qPKnSx4181RCnJvTQQc/9/YakV4eLCDYZlGythQCR
+2uk9gA4LpJ5eB+gFJmf/14gT8VCH1tJT7yReYegx6Fx9SpSTdK7AWpaSeTYhmQ9a
+PhnXWyIgXG3+01uygvXRXcez5nF42heAH3Q8sftWZRWokcGy+veFrpvBQNcZ/k1g
+Dbc1DzeT+WxN6c5PT5kpYVCbJR0ktnQe1ByMx7XWIiT0P6WKTYzufZu8Y3T6bjUN
+S7X1HhiJCDnncB4uByWYD+V9b/aKxgadFiAGkrh4LDRJXXFTDNWts2k07MmhtMeG
+MtfhkZ3HmJTBifQUqWgNG0JoPJ54bWlYm7L7Cn0MqemRNr1Ft/RvBfw8dYf447hM
+spOv7Fw7Fc8jn3f+GPrKHbr4B6IOqttnmFQoI4u3sTA6vaLcwkYEEBECAAYFAlDv
+uDsACgkQkVrMRaj0wv1SdgCeO+ktq5mAY5lZA3lA/I6hAKIk2s4An0L5XXjRvdHI
+xY5HSVNmRnyiIt9nwsL8BDABCAAmBQJS8CtrHx0AU2NyZXdlZCB1cCBlbWFpbCB2
+ZXJpZmljYXRpb24ACgkQWY+VqvbJnF9GxBv/QuxL5zlNM8jhzk5fuPpN9i++W88q
+fveAaBtxLTWe2UJl1/10xHfjw1a7FjJ3osYnEv21yW/w+nfhnMUk2DugC40/zb/+
+EaPrAIPsvyXDRjJNf89qYLqi4cagsPI5rzTy230iYk1kXrN3pyIY7feNd0dWpkCs
+DI5ug3hbezCGYLcg0B+4msMkuAGOU1Ctus/L2sIpdY277QZqFarK0xgYSGTZt/oh
+GXo3NbUakbELxeTgQSdLmeH6SchzES7Kv4Wanv3LRNAZHhV7wJSrAiuHpgDn6d/Y
+WqFwW2PoSTWN1CgCyZNOgn5cZTCTNnwo90USwyHid/x9oZlmMNALnWk93AOB1TBy
+OBoSbjfiK0xQAmrQlik+eOSYEeIxGg+B/CKy7wLqy5DEbf3IeNAiI1UVDQZb4woX
+J+Kx+gvcIOstK5AGzLXowaXdWrqsrfRQEXi6qxewv7P5hFC/e1/7645lLHyivtTj
+4F5peuQYgopvxKbjj/9ky6YgPsp77vdYCZRhU4xHyWH9zXZwVCdMe5GNKYcxMyJ9
+ATECmKxfGmXAiqFxtzRHroblCVH4ad5h+/dShPH+LUwVJorzp0gBx9E+ZycukMPm
+EVzji6CT8na7ptNoexCIt1TH2vwwFJwLPxlRGVEcj6aA7iq54Kwp/hpjYMbg11Qv
+Qpqw4Ug4tXs331IPkeYbKbaPudvfeBy3FucO5YSBFnKFIq77Q3ID500L7e9ttnvt
+n/s3ZIZqY5/87jv30x39o0SoAInsco3hViAYRGZrLKk4jAyM/q88YCa6r7kcPSLp
+YmfI+7OxFYuS/NvN46UL2ozm3NQzmXJKiAeCQiAVys89eI5V+QOotMz/hMRb96OE
+M/AAtrZLGXXxlRC0aJJYXXPYELWC0qMxvrR6Z/0LYyKssICH5Qgv7VbrFRLlVw64
+VV/MZjI/grBXP5yHarUPBruCOHe35bl0r7Sc+WAyqFgmiP8EXq8vwTQvBouEG3Tl
+7sScHgKNQ6+2Bhy3F/deh4afN+21xZ35I4HS8MZNaFSnQXktdKqSMxnpUhp8oFos
+hQZ1mz5Muq+Rdgk84LK1B+HxXqMaXz4pcF7SuvU8ZoeMlkbLBdLCAzgHUiL1PZ9u
+Ya8BxOcKErg46SakkkwbX90jEo2W2yv7zRuuKySPZaJ8VTHKQtzwuB+S5OmfE5WD
+zY/SHfQVGPZoKGPOwI0EUOshYwEKAMqU40j7kGpy7r37vZ+Ytk+LPMRSwhEDZjTD
+ZETv64nkSz39hOnk+dYA2k9PsZLwkmdzo0kl6HoaQyQYbCrk6nsIOyNb2lBnS8Bb
+3ReOfKeINr1bRb6bn5f8s87OH6eKz1lx/Xs/3W2mssIuL5M45vfnG3f3qln5L4/C
+5XR0uIhh1VhXd7os0JXQuOESqnndNHBOstM09BWe3QM9hOH8qfXHp3nM5LQwrhDJ
+so3VYlTqdghBFfJYqSLGNuz76NyBX+O5yT3pV7RuW+foN+p+kbxjNuapEK58ujrz
+cu2UFRnRz7OesPWei6pfYRv8LKUbxDxlQdeKYIn6DpF8f2Q6a1Uf/bTy7+cOh9Uv
+9DR28Bd9Tkxfj1ztdjLsHatOWT7ie415oczRpTZjXj5JDL6xHrPJ27t4Yt2qPNXQ
+Jf96SCuNABEBAAHCwLwEGAEIACYCGwwWIQQ0PC/w++5ewu2+85nzWZ/4KMZymAUC
+aF2UNAUJINh0UQAKCRDzWZ/4KMZymJwyCf0XEZBvA/hzq8V2wk2EMVysfoGDfTh8
+PEnK21Q4h5El685EKJNuMXR6IRlVtyhOlx4Yu6C8Po7HUisLlToNZ+F2+PeZFzio
+INZKu1FJ4fybkpHqPtwGcsW5molvYW9mUwLTrOstXppLoyhXlsuJRBl/AFs0ac07
+4NsbylM2421+lCB//Mye8sv9Gm14fq+Z1sHNmn4iHqM1MfMAU0WSCKnXDtkQmqXz
+KSOk7PzmgclGTq9Kg8jgmx1ixmzCaTqEe6KVM4IxtK/RCiZ1oE5Qph9CPZLH38su
+PcuXFy+z4WbvB6f1s8reOlrXa2eGTnSga8h7l8Sy7k8v4AdfrNeIc9RjDvxB9/uB
+BszPYSsPB7K4d37KkmyE9VLGL93r61vDgyerfLSkt3sB6O6L1dsMkePAQNjUmJzU
+CNXKoGy2n+llvsLAvAQYAQgAJgIbDBYhBDQ8L/D77l7C7b7zmfNZn/goxnKYBQJe
+Ie0PBQkWnM0sAAoJEPNZn/goxnKYUl4KAIgdXi/Q+ExT3ZtE5Z5lFQ+wwgmRKLnc
+0yrvO2d4MJNt+Sy/EZK3dbzIRFWPZukYRg26Y4ySilv8g3Fq5bai2afv+Wo3322h
+L1RavbpItIYbDQ47gj9AlWyY8IN4ux3AOI6m4Di29itG5h9G8nY/vzf2S1MxnngK
+PVcm12zgtSoQ9Vbx4UZfwhyqs/VqIsPNiC59Asuaj45PGEX2JBUHNGBzc8xXfJWv
+/Jeh3iZAkZZTS+3U3S19/p29ChDBRCzAyjlQPPkFLBtJCyn8nt8pGZrJBJbyDTk3
+GW6FcOiI+Hqvn1eAJGC+/y5zRTcG+7w+o3Z1nGIDBBo0QvhYSp9mrOjOOUpqiwLa
+9utimVoLYgc8BFyYObGKgo8l0WMMp0M6itctN6Od1Ww8rY1bqwH5uHdqQ23eUhdo
+yIu8B82gU2dawsC8BBgBAgAPBQJQ6yFjAhsMBQkSzAMAACEJEPNZn/goxnKYFiEE
+NDwv8PvuXsLtvvOZ81mf+CjGcpgZRwn+P4vT+7N/AJHVpMFnoUFP0503Jq1WObPb
+RqGyQXcHJKaRPVHTorTxg4Yn/8s9qOPR4qpTxYqDExzulaLlIb7IwWfZ1vkeckMa
+JKZKMJdb8HOxK3Xvzou07SbR6Fq2P2SXomvFA1drM6REnbDDXc/r/3VEFpXpHTwG
+YVOtRyjY/rD52MIXg2WeQ4trt8zpC01kz7IH4TNo+NIEWNkbnYFVZmRxcRU3RwSM
++7Oe5A6bHwOfLuVCKSnj026attqZrFkSKXHYGL2BJui81ooG6dwqL1duiISI1sBq
+sbE/jRC+WTYC5da+r5+IGdTV7WdTahXv/ucBYfuSP9Wv8nFwF46mw0MaRbtit3ea
+QkKuEb6MyzVXao1v1SGvlks4DjSwjPY8d8OtH6BzxMebank68JYQB66RgnbZdNQi
+3peg87QRFgM=
+=HQUb
+-----END PGP PUBLIC KEY BLOCK-----
diff --git a/sources b/sources
index 7eb563d..5f5ddb1 100644
--- a/sources
+++ b/sources
@@ -1,3 +1,2 @@
 SHA512 (nettle-3.10.2.tar.gz) = bf37ddd7dca8e78488da2a5286dcf16761d527d620572b42f2ad27bb8ee8c12999d92b0272e06f53766e7155a3f4a1ab7ad9c4b1c3caec47c031878b6b1772fb
 SHA512 (nettle-3.10.2.tar.gz.sig) = 82d82ba83b65b191d0ad9af24603df8dbc1b4172118618be66c267e51bad470d902e7194de1948e3570f144e0021afb7ddfb81badbe71fd57999d3d3227f0d39
-SHA512 (nettle-release-keyring.gpg) = 0e59447eb74017439c8b5b5b05173c0ffd710705d2a9c1f74833b7034fad1608fa1bdd2c308e6c42214553cd648606b6a07044ea39677b1b3452cb4d07bf889b
-- 
2.56.0


